### DISCUSSION TITLE
VS code ssh remote notebook experience slow response when there is a intense display requirement

### DISCUSSION THREAD

**Discussion by yingruiz-github** [microsoft/vscode-discussions#2441] [Extension Development QnA]
Hi, 

This is the first time that I use this discussion thread so please let me know if this question is off topic.

I found vs code notebook remote SSH not working properly when notebook need to display a lot of things. For example, when I use matplotlib to draw some very high resolution pictures, the notebook will stop from coloring the variables (first one is what the code looks normally, and the bottom one is when I display a lot of high resolution images in the notebook)
![image](https://github.com/user-attachments/assets/c1f5355d-b8e7-464f-a324-2fb32bc9e25e)
![image](https://github.com/user-attachments/assets/46c8115e-6836-4aa0-8db6-e6dcd2067fba)

Or trying running a cell will result the cell frozen like following (note, I have click the run button, but as you can see, the cell frozened)
 <img width="1242" alt="image" src="https://github.com/user-attachments/assets/5455976c-b2c8-4852-bb0c-606f9d4cc353" />
 
 Also, sometimes showing a lot of high resolution images can cause the notebook became very laggy and "interrupt" and "restart" button will stop working.
 
 I wonder does vs code team aware of this issue? if yes, is there a solution for the problem?
 
 Thanks

**Comment by olguzzar** [comment]
Hi @yingruiz-github! Thanks so much for posting. Since this is related to remote SSH in Jupyter notebooks in VS Code, the [vscode-jupyter repo](https://github.com/microsoft/vscode-jupyter) or the [vscode-remote-release](https://github.com/microsoft/vscode-remote-release) repo is better suited for this.

This forum is specifically for extension authors actively developing their own extensions, so I'm going to close this as off-topic.

**Reply by yingruiz-github** [reply]
Thanks
